chandra_aca/maude_decom.py

```diff
@@ -0,0 +1,485 @@
+
+    pixels = [[f'{msid_prefix}CIMG{px_img_num}{px_id}{px_num}' for px_num in px_nums for px_id in px_ids]
+              for px_img_num in px_img_nums]
+
+    res = {
+        'pixels': pixels,
+        'sizes': [f'{msid_prefix}CA00040',   # Size of image 0
+                  f'{msid_prefix}CA00043',   # Size of image 1
+                  f'{msid_prefix}CA00046',   # Size of image 2
+                  f'{msid_prefix}CA00049',   # Size of image 3
+                  f'{msid_prefix}CA00052',   # Size of image 4
+                  f'{msid_prefix}CA00055',   # Size of image 5
+                  f'{msid_prefix}CA00058',   # Size of image 6
+                  f'{msid_prefix}CA00061'],   # Size of image 7
+
+        'rows': [f'{msid_prefix}CA00076',   # Row of pixel A1 of image 0
+                 f'{msid_prefix}CA00292',   # Row of pixel A1 of image 1
+                 f'{msid_prefix}CA00508',   # Row of pixel A1 of image 2
+                 f'{msid_prefix}CA00724',   # Row of pixel A1 of image 3
+                 f'{msid_prefix}CA00940',   # Row of pixel A1 of image 4
+                 f'{msid_prefix}CA01156',   # Row of pixel A1 of image 5
+                 f'{msid_prefix}CA01372',   # Row of pixel A1 of image 6
+                 f'{msid_prefix}CA01588'],   # Row of pixel A1 of image 7
+
+        'cols': [f'{msid_prefix}CA00086',   # Column of pixel A1 of image 0
+                 f'{msid_prefix}CA00302',   # Column of pixel A1 of image 1
+                 f'{msid_prefix}CA00518',   # Column of pixel A1 of image 2
+                 f'{msid_prefix}CA00734',   # Column of pixel A1 of image 3
+                 f'{msid_prefix}CA00950',   # Column of pixel A1 of image 4
+                 f'{msid_prefix}CA01166',   # Column of pixel A1 of image 5
+                 f'{msid_prefix}CA01382',   # Column of pixel A1 of image 6
+                 f'{msid_prefix}CA01598'],  # Column of pixel A1 of image 7
+
+        'scale_factor': [f'{msid_prefix}CA00096',   # Scale factor of image 0
+                         f'{msid_prefix}CA00312',   # Scale factor of image 1
+                         f'{msid_prefix}CA00528',   # Scale factor of image 2
+                         f'{msid_prefix}CA00744',   # Scale factor of image 3
+                         f'{msid_prefix}CA00960',   # Scale factor of image 4
+                         f'{msid_prefix}CA01176',   # Scale factor of image 5
+                         f'{msid_prefix}CA01392',   # Scale factor of image 6
+                         f'{msid_prefix}CA01608'],  # Scale factor of image 7
+
+        'image_status': [f'AOIMAGE{i}' for i in range(8)],    # IMAGE 0 STATUS FLAG
+        'fiducial_flag': [f'AOACFID0{i}' for i in range(8)],  # FIDUCIAL LIGHT FLAG (OBC)
+        'image_function': [f'AOACFCT{i}' for i in range(8)],  # IMAGE FUNCTION (OBC)
+        # this one exists also as FUNCTION2/3/4
+        #'image_function_pea':
+        #    [f'{msid_prefix}AIMGF{i}1' for i in range(8)],  # IMAGE FUNCTION1 (PEA)
```

Comment:
Do the time stamps from the OBC AOACFCT image function map as well onto the pixel data as the pea image function (are they the same)?

Reply:
no idea, I haven't looked at that yet.